README.md

```diff
@@ -167,7 +167,7 @@ Some features are currently **not supported**:
 * Some experimental syntax extensions (e.g. decorators).
 * CSS Modules.
 * LESS or Sass.
-* Hot reloading of components.
+* Hot reloading. Full page reload is triggered when you change a file.
```

Comment:
I’m not sure what you mean. The list is “Limitations”, that is, features that are *not* supported. Hot reloading of components is not supported, so it should be on the list.